### PR TITLE
changelog: prepare for v2.3.0 release

### PR DESCRIPTION
**Pull request by 4o4x**

### CODE REVIEW

**Review by seruman — DISMISSED** [review]

**Review by seruman — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
 # Changelog
-## v2.3.0
+## v2.3.0 - 16 December 2024
 
 #### Breaking changes
-- Changed the exit code from 1 to 0 for `ls` when used with an empty bucket. Exits with 1 if the bucket is non-existent. ([#722](https://github.com/peak/s5cmd/issues/722))
+- Changed the exit code from 1 to 0 for `ls` when used with an empty bucket. Exits with 1 if the bucket is non-existent. ([#722](https://github.com/peak/s5cmd/issues/722)) [@Z9n2JktHlZDmlhSvqc9X2MmL3BwQG7tk](https://github.com/Z9n2JktHlZDmlhSvqc9X2MmL3BwQG7tk)
+
+#### Improvements
```
Comment (igungor) [inline CHANGELOG.md:7]: Can you order the subsections as Feaures -> Improvements -> Bug Fixes please?

Comment (4o4x) [reply CHANGELOG.md]: Done 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
 # Changelog
-## v2.3.0
+## v2.3.0 - 16 December 2024
```
Comment (seruman) [inline CHANGELOG.md:2]: I've missed this one previously; like previous entries, let's use short form
```suggestion
## v2.3.0 - 16 Dec 2024
```

Comment (4o4x) [reply CHANGELOG.md]: Done 👍

**Review by igungor — APPROVED** [review]

**Review by seruman — APPROVED** [review]

**Pull request merged**